packages/accounts-2fa/2fa-server.js

```diff
@@ -0,0 +1,95 @@
+import { Accounts } from 'meteor/accounts-base';
+import twofactor from "node-2fa";
+import QRCode from "qrcode-svg";
+import { Meteor } from "meteor/meteor";
+
+Accounts.checkUserHas2FAEnabled = selector => {
+  if (!Meteor.isServer) {
+    throw new Meteor.Error(400, "The function checkUserHas2FAEnabled can only be called on the server");
+  }
+
+  if (typeof selector === 'string') {
+    if (!selector.includes('@')) {
```

Comment:
I think this logic is not new, correct? Other packages have already decided this using this simple rule, am I right?

Reply:
Yes, other methods use this strategy. It wasn't something I invented.